CSS/media.css

```diff
@@ -90,4 +88,53 @@
   .menu {
     display: none;
   }
+
+  .pop-content {
+    margin-left: 250px;
+    padding: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    width: 650px;
+    height: 650px;
+    background: #fff;
+    border: 1px solid #ebecf0;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+    border-radius: 16px;
+  }
+
+  .close-icon {
+    right: 30px;
+    cursor: pointer;
+  }
+
+  .popimg {
+    display: flex;
+    position: relative;
+    width: 620px;
+    height: 380px;
+    margin-bottom: 40px;
+  }
+
+  .bottom {
+    display: flex;
+    margin-bottom: 40px;
+  }
+
+  .bottom > p {
+    line-height: 30px;
+  }
+
+  .right-side {
+    margin-left: 30px;
+  }
+
+  .right-side > .project-languages {
+    margin-bottom: 20px;
+  }
+
+  .popup-bottom {
+    display: flex;
+    margin-bottom: 40px;
+  }
```

Comment:
- [ ] Your popup is not centered in some screen sizes and the image, buttons and tags are a little bit off from the Figma design. Kindly fix the heights, margins, paddings, etc...
Your desktop design:
<img width="937" alt="image" src="https://user-images.githubusercontent.com/61919617/181077157-7eb9776b-6f5e-4f6b-a118-a69b82a82b38.png">